test/jasmine/tests/annotations_test.js

```diff
@@ -1127,7 +1132,7 @@ describe('annotation effects', function() {
         }
 
         makePlot([
-            {x: 50, y: 50, text: 'hi', width: 50, ax: 0, ay: -40},
+            {x: 50, y: 50, text: 'hi', width: 50, ax: 0, ay: -40, xshift: -50, yshift: 50},
```

Comment:
Is `editable: true` tested with xshift / yshift? It's a little hard to tell from the diff.